creed/file_hash.py

```diff
@@ -0,0 +1,48 @@
+# adapted from https://stackoverflow.com/questions/3431825/generating-an-md5-checksum-of-a-file
+import hashlib
+import os
+from ark.utils import io_utils, misc_utils
+
+def get_hash(filepath):
+    """Computes the hash of the specified file to verify file integrity
+
+    Args:
+        filepath: full path to file
+
+    Returns:
+        string: the hash of the file"""
+
+    with open(filepath, "rb") as f:
+        file_hash = hashlib.blake2b()
+        while chunk := f.read(8192):
+            file_hash.update(chunk)
+    return  file_hash.hexdigest()
+
+
+def compare_directories(dir_1, dir_2):
```

Comment:
quick q here.  is the intention to compare hashes between directories on the CAC and those on an external drive, or...?

Reply:
Exactly. We copy a couple TBs off the CAC, want to ensure it went smoothly